app/views/components/_important-metadata.html.erb

```diff
@@ -0,0 +1,12 @@
+<%
+    items = local_assigns[:items] || {}
+    items = items.reject { |k,v| v.nil? }
+    items = items.merge(items) { |k,v| Array(v).join(",").html_safe }
```

Comment:
Nitpick: are these lines indented too far?

Reply:
Fixed in https://github.com/alphagov/government-frontend/pull/541/commits/58e7747dc20e08c7a3b9518091b2a60c5e16d071